.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -77,6 +77,7 @@ runs:
             'avm/res/azure-stack-hci/cluster'                    # Failing on resource deletion when trying to delete RBAC at subscription level
             'avm/res/azure-stack-hci/logical-network'            # Failing on resource deletion when trying to delete RBAC at subscription level
             'avm/res/azure-stack-hci/network-interface'          # Failing on resource deletion when trying to delete RBAC at subscription level
+            'avm/res/azure-stack-hci/virtual-hard-disk'          # Failing on resource deletion when trying to delete RBAC at subscription level
```

Comment:
We'd need to update the ci separately from the PR, to avoid triggering all module workflows on this change. I'll have that done in a separate PR and let you know once ready to pull latest from upstream

Reply:
This was done